bidiUtils.js

```diff
@@ -26,6 +26,5 @@ function baseTextDir(dir,str,defaultdDir){
     }
   }
   return defaultDir || "ltr";
- 
-  }
+ }
```

Comment:
With ES6 (http://es6-features.org/#DefaultParameterValues), you can use default parameters like this baseTextDir(dir, str, defaultDir = 'ltr') instead of returning with OR